DragaliaAPI.Database/Entities/DbSetUnit.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using DragaliaAPI.Shared.Definitions.Enums;
+
+namespace DragaliaAPI.Database.Entities;
+
+//TODO: This and PartyUnit share a lot of properties, maybe extract those and make these into subclasses which inherit them
+[Table("PlayerSetUnit")]
+public class DbSetUnit : IDbHasAccountId
+{
+    [Required]
+    [ForeignKey("DbDeviceAccountId")]
+    public string DeviceAccountId { get; set; } = null!;
```

Comment:
In C#11 we can now do `public required string` to shut the compiler up instead of using `null!`